certified-connectors/Plumsail Forms/apiDefinition.swagger.json

```diff
@@ -18,6 +18,75 @@
   "consumes": [],
   "produces": [],
   "paths": {
+    "/api/attachments": {
+      "get": {
+        "tags": [
+          "Attachments"
+        ],
+        "operationId": "DownloadAttachment",
+        "consumes": [],
+        "produces": [
+          "application/octet-stream"
+        ],
+        "parameters": [
+          {
+            "name": "fileUrl",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "File URL",
+            "description": "File URL"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Result file",
+            "schema": {
+              "format": "binary",
+              "title": "Result file",
+              "description": "",
```

Comment:
@jusper-dk Please provide valid description value for all the empty properties.

Reply:
Done!